.travis.yml

```diff
@@ -32,11 +32,15 @@ jobs:
         - docker tag bitbots/bitbots-base:kinetic bitbots/bitbots-base:latest
         - docker push bitbots/bitbots-base:latest
     - stage: deploy
-      name: "Domestic: base image"
+      name: "Domestic and Industrial: base image"
```

Comment:
I would suggest we build the domestic and industrial images separately. To do that just add a new element to the list of jobs please.

Reply:
Done